main.tf

```diff
@@ -1,59 +1,165 @@
-module "lambda" {
-  source                         = "./modules/lambda"
+  module "lambda" {
+  source                         = "app.terraform.io/Bankrate/lambda-function/aws"
+  version                        = "~> 3.0.0" # Only pull patch/fix releases
   description                    = var.description
-  environment                    = var.environment
   filename                       = var.filename
-  function_name                  = var.function_name
   handler                        = var.handler
-  memory_size                    = var.memory_size
   publish                        = var.publish
   reserved_concurrent_executions = var.reserved_concurrent_executions
   runtime                        = var.runtime
   timeout                        = var.timeout
   tags                           = var.tags
   vpc_config                     = var.vpc_config
   layers                         = var.layers
+
+ # additions from RV standard
+  name                = var.function_name
```

Comment:
What's the difference between `app_name`, `function_name`, and `name`? Could we combine these some how to reduce the variables required from users?